invenio.cfg

```diff
@@ -274,9 +274,12 @@ USERPROFILES_EXTEND_SECURITY_FORMS = True  # automatically use user's email addr
 
 # OAI-PMH
 # =======
-OAISERVER_ID_PREFIX = "cds-rdm.com"
+OAISERVER_ID_PREFIX = "repository.cern"
 """The prefix that will be applied to the generated OAI-PMH ids."""
 
+OAISERVER_ADMIN_EMAILS = ["cds-team@cern.ch"]
```

Comment:
thanks for finding this one too!
the default should be "CHANGE_ME" since we don't want to expose our emails in the GH repos if not necessary (worring about spam)